backend/src/common/types/controller/controller.ts

```diff
@@ -1,26 +1,32 @@
+import { TokenPayload } from '@autoline/shared';
 import { Request } from 'express';
 import { Query, ParamsDictionary } from 'express-serve-static-core';
 
-interface TypedRequestBody<T> extends Request {
+interface TypedRequestBody<T> extends AuthRequest {
   body: T;
 }
 
-interface TypedRequestQuery<T extends Query> extends Request {
+interface TypedRequestQuery<T extends Query> extends AuthRequest {
   query: T;
 }
 
-interface TypedRequestParams<T extends ParamsDictionary> extends Request {
+interface TypedRequestParams<T extends ParamsDictionary> extends AuthRequest {
   params: T;
 }
 
-interface TypedRequest<T extends Query, U> extends Request {
+interface TypedRequest<T extends Query, U> extends AuthRequest {
   body: U;
   query: T;
 }
 
+interface AuthRequest extends Request {
+  tokenPayload?: TokenPayload;
+}
```

Comment:
Make the `tokenPayload` defined. Then devs will be able access it w/o `as` in places where it's expected